docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>nit: passing a parameter into target -&gt; passing a parameter into <code>target</code></p><p> </p><p> <code>target</code>  doesn't matter when <code>pps</code> is True because <code>pps=True</code> means to plot the posterior predictive distribution and that can only refer to the outcome variable (instead of any of the parameters of the response distribution).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Yeah, I like the further clarification here.</p>